Index: app/src/main/java/com/djonique/birdays/activities/DetailActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/activities/DetailActivity.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/DetailActivity.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -1,4 +1,4 @@
-package com.djonique.birdays.activities;
+package com.djonique.birdays;
 
 import android.content.Intent;
 import android.os.Bundle;
@@ -10,9 +10,7 @@
 import android.view.MenuItem;
 import android.widget.TextView;
 
-import com.djonique.birdays.R;
 import com.djonique.birdays.adapters.AllFragmentAdapter;
-import com.djonique.birdays.utils.Utils;
 
 public class DetailActivity extends AppCompatActivity {
 
Index: app/src/main/java/com/djonique/birdays/activities/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/activities/MainActivity.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/MainActivity.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -1,4 +1,4 @@
-package com.djonique.birdays.activities;
+package com.djonique.birdays;
 
 import android.app.DialogFragment;
 import android.app.FragmentManager;
@@ -14,7 +14,6 @@
 import android.view.View;
 import android.widget.Toast;
 
-import com.djonique.birdays.R;
 import com.djonique.birdays.adapters.PagerAdapter;
 import com.djonique.birdays.database.DBHelper;
 import com.djonique.birdays.dialogs.NewPersonDialogFragment;
@@ -22,12 +21,11 @@
 import com.djonique.birdays.fragments.FamousFragment;
 import com.djonique.birdays.fragments.MonthFragment;
 import com.djonique.birdays.model.Person;
-import com.djonique.birdays.utils.ConstantManager;
 
-
 public class MainActivity extends AppCompatActivity implements
         NewPersonDialogFragment.AddingPersonListener, AllFragment.DeletingRecordListener {
 
+    public static final String NEW_PERSON_DIALOG_TAG = "NewPersonDialogFragment";
     public DBHelper dbHelper;
     MonthFragment monthFragment;
     AllFragment allFragment;
@@ -109,7 +107,7 @@
             @Override
             public void onClick(View v) {
                 DialogFragment newPersonDialogFragment = new NewPersonDialogFragment();
-                newPersonDialogFragment.show(fragmentManager, ConstantManager.NEW_PERSON_DIALOG_TAG);
+                newPersonDialogFragment.show(fragmentManager, NEW_PERSON_DIALOG_TAG);
             }
         });
     }
Index: app/src/main/java/com/djonique/birdays/adapters/AllFragmentAdapter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/adapters/AllFragmentAdapter.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/adapters/AllFragmentAdapter.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -11,9 +11,9 @@
 import android.view.ViewGroup;
 import android.widget.TextView;
 
-import com.djonique.birdays.activities.DetailActivity;
+import com.djonique.birdays.DetailActivity;
 import com.djonique.birdays.R;
-import com.djonique.birdays.utils.Utils;
+import com.djonique.birdays.Utils;
 import com.djonique.birdays.fragments.AllFragment;
 import com.djonique.birdays.model.Item;
 import com.djonique.birdays.model.Person;
Index: app/src/main/java/com/djonique/birdays/adapters/FamousFragmentAdapter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/adapters/FamousFragmentAdapter.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/adapters/FamousFragmentAdapter.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -10,7 +10,7 @@
 import android.widget.TextView;
 
 import com.djonique.birdays.R;
-import com.djonique.birdays.utils.Utils;
+import com.djonique.birdays.Utils;
 import com.djonique.birdays.model.Item;
 import com.djonique.birdays.model.Person;
 
Index: app/src/main/java/com/djonique/birdays/adapters/MonthFragmentAdapter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/adapters/MonthFragmentAdapter.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/adapters/MonthFragmentAdapter.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -13,19 +13,25 @@
 import android.widget.TextView;
 
 import com.djonique.birdays.R;
+import com.djonique.birdays.Utils;
 import com.djonique.birdays.model.Item;
 import com.djonique.birdays.model.Person;
-import com.djonique.birdays.utils.ConstantManager;
-import com.djonique.birdays.utils.Utils;
 
 import java.util.ArrayList;
 import java.util.List;
 
-
 public class MonthFragmentAdapter extends RecyclerView.Adapter<MonthFragmentAdapter.CardViewHolder> {
 
+    private static final String TYPE_EMAIL = "message/rfc822";
+    private static final String MAILTO = "mailto:";
+    private static final String TYPE_SMS = "vnd.android-dir/mms-sms";
+    private static final String ADDRESS = "address";
+    private static final String SMSTO = "smsto:";
+    private static final String TEL = "tel: ";
+
     private Context context;
     private List<Item> items;
+
     private int disabled = Color.rgb(224, 224, 224);
     private int enabled = Color.rgb(156, 39, 176);
 
@@ -79,7 +85,7 @@
             holder.btnPhone.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    Intent intent = new Intent(Intent.ACTION_DIAL, Uri.parse(ConstantManager.TEL + person.getPhoneNumber()));
+                    Intent intent = new Intent(Intent.ACTION_DIAL, Uri.parse(TEL + person.getPhoneNumber()));
                     context.startActivity(intent);
                 }
             });
@@ -89,9 +95,9 @@
                 public void onClick(View v) {
                     long phoneNumber = person.getPhoneNumber();
                     Intent intent = new Intent(Intent.ACTION_VIEW);
-                    intent.setType(ConstantManager.TYPE_SMS);
-                    intent.putExtra(ConstantManager.ADDRESS, phoneNumber);
-                    intent.setData(Uri.parse(ConstantManager.SMSTO + phoneNumber));
+                    intent.setType(TYPE_SMS);
+                    intent.putExtra(ADDRESS, phoneNumber);
+                    intent.setData(Uri.parse(SMSTO + phoneNumber));
                     context.startActivity(intent);
                 }
             });
@@ -112,10 +118,10 @@
                 public void onClick(View v) {
                     String email = person.getEmail();
                     Intent intent = new Intent(Intent.ACTION_SENDTO);
-                    intent.setType(ConstantManager.TYPE_EMAIL);
+                    intent.setType(TYPE_EMAIL);
                     intent.putExtra(Intent.EXTRA_EMAIL, new String[]{email});
                     intent.putExtra(Intent.EXTRA_SUBJECT, context.getString(R.string.happy_birthday));
-                    intent.setData(Uri.parse(ConstantManager.MAILTO + email));
+                    intent.setData(Uri.parse(MAILTO + email));
                     context.startActivity(Intent.createChooser(intent, context.getString(R.string.send_email)));
                 }
             });
Index: app/src/main/java/com/djonique/birdays/behaviors/ScrollAwareFABBehavior.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/behaviors/ScrollAwareFABBehavior.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/ScrollAwareFABBehavior.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -1,4 +1,4 @@
-package com.djonique.birdays.behaviors;
+package com.djonique.birdays;
 
 import android.content.Context;
 import android.support.design.widget.CoordinatorLayout;
Index: app/src/main/java/com/djonique/birdays/database/DBQueryManager.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/database/DBQueryManager.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/database/DBQueryManager.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -3,7 +3,7 @@
 import android.database.Cursor;
 import android.database.sqlite.SQLiteDatabase;
 
-import com.djonique.birdays.utils.Utils;
+import com.djonique.birdays.Utils;
 import com.djonique.birdays.model.Person;
 
 import java.util.ArrayList;
Index: app/src/main/java/com/djonique/birdays/dialogs/NewPersonDialogFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/dialogs/NewPersonDialogFragment.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/dialogs/NewPersonDialogFragment.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -1,22 +1,12 @@
 package com.djonique.birdays.dialogs;
 
-import android.Manifest;
 import android.app.Activity;
 import android.app.AlertDialog;
 import android.app.Dialog;
 import android.app.DialogFragment;
-import android.content.ContentResolver;
-import android.content.Context;
 import android.content.DialogInterface;
-import android.content.Intent;
-import android.content.pm.PackageManager;
-import android.database.Cursor;
-import android.net.Uri;
 import android.os.Bundle;
-import android.provider.ContactsContract;
 import android.support.design.widget.TextInputLayout;
-import android.support.v4.app.ActivityCompat;
-import android.support.v4.content.ContextCompat;
 import android.text.Editable;
 import android.text.TextUtils;
 import android.text.TextWatcher;
@@ -26,27 +16,25 @@
 import android.widget.EditText;
 
 import com.djonique.birdays.R;
+import com.djonique.birdays.Utils;
 import com.djonique.birdays.model.Person;
-import com.djonique.birdays.utils.ConstantManager;
-import com.djonique.birdays.utils.Utils;
 
 import java.util.Calendar;
 
-
-
 public class NewPersonDialogFragment extends DialogFragment implements
         com.wdullaer.materialdatetimepicker.date.DatePickerDialog.OnDateSetListener {
 
-    private EditText etPhone, etEmail, etDate;
+    public static final String DATE_PICKER_FRAGMENT_TAG = "DatePickerFragment";
+    private EditText etDate;
     private Calendar calendar;
     private AddingPersonListener addingPersonListener;
 
     @Override
-    public void onAttach(Context context) {
-        super.onAttach(context);
+    public void onAttach(Activity activity) {
+        super.onAttach(activity);
 
         try {
-            addingPersonListener = (AddingPersonListener) getActivity();
+            addingPersonListener = (AddingPersonListener) activity;
         } catch (ClassCastException e) {
             throw new ClassCastException();
         }
@@ -68,28 +56,10 @@
         final EditText etName = tilName.getEditText();
 
         final TextInputLayout tilPhone = (TextInputLayout) container.findViewById(R.id.tilPhone);
-        etPhone = tilPhone.getEditText();
+        final EditText etPhone = tilPhone.getEditText();
 
-        Button addFromContactsButton =
-                ((Button) container.findViewById(R.id.addFromContactsDialogButton));
-        addFromContactsButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                if (ContextCompat.checkSelfPermission(getActivity(),
-                        Manifest.permission.READ_CONTACTS) ==
-                        PackageManager.PERMISSION_GRANTED) {
-                    Intent intent = new Intent(Intent.ACTION_PICK, ContactsContract.Contacts.CONTENT_URI);
-                    startActivityForResult(intent, ConstantManager.REQUEST_READ_CONTACTS);
-                } else {
-                    ActivityCompat.requestPermissions(getActivity(),
-                            new String[]{Manifest.permission.READ_CONTACTS},
-                            ConstantManager.CONTACTS_REQUEST_PERMISSION_CODE);
-                }
-            }
-        });
-
         final TextInputLayout tilEmail = (TextInputLayout) container.findViewById(R.id.tilEmail);
-        etEmail = tilEmail.getEditText();
+        final EditText etEmail = tilEmail.getEditText();
 
         final TextInputLayout tilDate = ((TextInputLayout) container.findViewById(R.id.tilDate));
         etDate = (EditText) container.findViewById(R.id.etDate);
@@ -113,7 +83,7 @@
                                 calendar.get(Calendar.DAY_OF_MONTH)
                         );
 
-                dpd.show(getFragmentManager(), ConstantManager.DATE_PICKER_FRAGMENT_TAG);
+                dpd.show(getFragmentManager(), DATE_PICKER_FRAGMENT_TAG);
             }
         });
 
@@ -133,7 +103,6 @@
                 }
 
                 if (etPhone != null && etPhone.length() != 0) {
-                    // TODO: 21.12.2016 БД должна хранить телефон в стринге
                     person.setPhoneNumber(Long.parseLong(etPhone.getText().toString()));
                 } else person.setPhoneNumber(0);
 
@@ -236,59 +205,6 @@
     private boolean isRightDate() {
         long today = Calendar.getInstance().getTimeInMillis();
         return today >= calendar.getTimeInMillis();
-    }
-
-    @Override
-    public void onActivityResult(int requestCode, int resultCode, Intent data) {
-        super.onActivityResult(requestCode, resultCode, data);
-        if (resultCode == Activity.RESULT_OK) {
-            Uri contactData = data.getData();
-            ContentResolver contentResolver = getActivity().getContentResolver();
-            Cursor cursor = contentResolver.query(contactData, null, null, null, null);
-            if (cursor != null && cursor.getCount() > 0) {
-                if (cursor.moveToFirst()) {
-                    String id = cursor.getString(
-                            cursor.getColumnIndex(ContactsContract.Contacts._ID));
-                    etPhone.setText(getPhoneFromContacts(contentResolver, cursor, id));
-                    etEmail.setText(getEmailFromContacts(contentResolver, id));
-                }
-            }
-            cursor.close();
-        }
-    }
-
-    // Retrieves phone number from picked contact
-    private String getPhoneFromContacts(ContentResolver contentResolver, Cursor cursor, String id) {
-        String phoneNumber = "";
-        if (Integer.parseInt(cursor.getString(
-                cursor.getColumnIndex(ContactsContract.Contacts.HAS_PHONE_NUMBER))) > 0) {
-            Cursor phoneCursor = contentResolver.query(
-                    ContactsContract.CommonDataKinds.Phone.CONTENT_URI,
-                    null,
-                    ContactsContract.CommonDataKinds.Phone.CONTACT_ID + " = ?",
-                    new String[]{id}, null);
-            if (phoneCursor.moveToFirst()) {
-                phoneNumber = phoneCursor.getString(phoneCursor.getColumnIndex
-                        (ContactsContract.CommonDataKinds.Phone.NUMBER));
-            }
-            phoneCursor.close();
-        }
-        return phoneNumber;
-    }
-
-    // Retrieves email from picked contact
-    private String getEmailFromContacts(ContentResolver contentResolver, String id) {
-        String email = "";
-        Cursor emailCursor = contentResolver.query(ContactsContract.CommonDataKinds.Email.CONTENT_URI,
-                null,
-                ContactsContract.CommonDataKinds.Email.CONTACT_ID + " = ?",
-                new String[]{id}, null);
-        if (emailCursor.moveToFirst()) {
-            email = emailCursor.getString(
-                    emailCursor.getColumnIndexOrThrow(ContactsContract.CommonDataKinds.Email.DATA));
-        }
-        emailCursor.close();
-        return email;
     }
 
     public interface AddingPersonListener {
Index: app/src/main/java/com/djonique/birdays/fragments/AllFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/fragments/AllFragment.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/fragments/AllFragment.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -13,7 +13,7 @@
 import android.view.View;
 import android.view.ViewGroup;
 
-import com.djonique.birdays.activities.MainActivity;
+import com.djonique.birdays.MainActivity;
 import com.djonique.birdays.R;
 import com.djonique.birdays.adapters.AllFragmentAdapter;
 import com.djonique.birdays.database.DBHelper;
Index: app/src/main/java/com/djonique/birdays/fragments/FamousFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/fragments/FamousFragment.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/fragments/FamousFragment.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -11,7 +11,7 @@
 import android.view.View;
 import android.view.ViewGroup;
 
-import com.djonique.birdays.activities.MainActivity;
+import com.djonique.birdays.MainActivity;
 import com.djonique.birdays.R;
 import com.djonique.birdays.adapters.FamousFragmentAdapter;
 import com.djonique.birdays.model.Person;
Index: app/src/main/java/com/djonique/birdays/fragments/MonthFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/fragments/MonthFragment.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/fragments/MonthFragment.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -9,7 +9,7 @@
 import android.view.View;
 import android.view.ViewGroup;
 
-import com.djonique.birdays.activities.MainActivity;
+import com.djonique.birdays.MainActivity;
 import com.djonique.birdays.R;
 import com.djonique.birdays.adapters.MonthFragmentAdapter;
 import com.djonique.birdays.database.DBHelper;
Index: app/src/main/java/com/djonique/birdays/utils/ConstantManager.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/utils/ConstantManager.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/utils/ConstantManager.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
@@ -1,17 +0,0 @@
-package com.djonique.birdays.utils;
-
-
-public interface ConstantManager {
-    String NEW_PERSON_DIALOG_TAG = "NEW_PERSON_DIALOG_TAG";
-    String DATE_PICKER_FRAGMENT_TAG = "DATE_PICKER_FRAGMENT_TAG";
-
-    String TYPE_EMAIL = "message/rfc822";
-    String MAILTO = "mailto:";
-    String TYPE_SMS = "vnd.android-dir/mms-sms";
-    String ADDRESS = "address";
-    String SMSTO = "smsto:";
-    String TEL = "tel: ";
-
-    int REQUEST_READ_CONTACTS = 1;
-    int CONTACTS_REQUEST_PERMISSION_CODE = 2;
-}
Index: app/src/main/java/com/djonique/birdays/utils/Utils.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/djonique/birdays/utils/Utils.java	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/java/com/djonique/birdays/Utils.java	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -1,6 +1,4 @@
-package com.djonique.birdays.utils;
-
-import com.djonique.birdays.R;
+package com.djonique.birdays;
 
 import java.text.DateFormat;
 import java.util.Calendar;
Index: app/src/main/res/drawable/divider.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/drawable/divider.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/drawable/divider.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -3,5 +3,5 @@
     android:shape="line">
     <stroke
         android:width="1px"
-        android:color="@color/divider_color" />
+        android:color="@color/dividerColor" />
 </shape>
\ No newline at end of file
Index: app/src/main/res/drawable/ic_contacts_purple_24dp.xml
===================================================================
--- app/src/main/res/drawable/ic_contacts_purple_24dp.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
+++ app/src/main/res/drawable/ic_contacts_purple_24dp.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:height="24dp"
+        android:width="24dp"
+        android:viewportWidth="24"
+        android:viewportHeight="24">
+    <path
+        android:fillColor="#9C27B0"
+        android:pathData="M20,0H4V2H20V0M4,24H20V22H4V24M20,4H4A2,2 0 0,0 2,6V18A2,2 0 0,0 4,20H20A2,2 0 0,0 22,18V6A2,2 0 0,0 20,4M12,6.75A2.25,2.25 0 0,1 14.25,9A2.25,2.25 0 0,1 12,11.25A2.25,2.25 0 0,1 9.75,9A2.25,2.25 0 0,1 12,6.75M17,17H7V15.5C7,13.83 10.33,13 12,13C13.67,13 17,13.83 17,15.5V17Z" />
+</vector>
\ No newline at end of file
Index: app/src/main/res/layout/activity_detail.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_detail.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/layout/activity_detail.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -14,7 +14,7 @@
             android:id="@+id/rootRelative"
             android:layout_width="match_parent"
             android:layout_height="192dp"
-            android:background="@color/color_primary"
+            android:background="@color/colorPrimary"
             android:orientation="vertical"
             android:paddingBottom="8dp"
             android:paddingTop="48dp">
@@ -148,7 +148,7 @@
                 android:layout_marginLeft="72dp"
                 android:layout_marginStart="72dp"
                 android:layout_marginTop="8dp"
-                android:background="@color/divider_color"/>
+                android:background="@color/dividerColor"/>
 
             <RelativeLayout
                 android:id="@+id/rlDate"
\ No newline at end of file
Index: app/src/main/res/layout/activity_main.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_main.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/layout/activity_main.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -7,7 +7,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fitsSystemWindows="true"
-    tools:context="com.djonique.birdays.activities.MainActivity">
+    tools:context="com.djonique.birdays.MainActivity">
 
     <android.support.design.widget.AppBarLayout
         android:id="@+id/appbar"
@@ -56,6 +56,6 @@
         android:layout_margin="@dimen/fab_margin"
         android:elevation="6dp"
         android:src="@drawable/ic_add_white_24dp"
-        app:layout_behavior="com.djonique.birdays.behaviors.ScrollAwareFABBehavior"/>
+        app:layout_behavior="com.djonique.birdays.ScrollAwareFABBehavior"/>
 
 </android.support.design.widget.CoordinatorLayout>
Index: app/src/main/res/layout/description_card_view.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/description_card_view.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/layout/description_card_view.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -44,7 +44,7 @@
             android:layout_width="match_parent"
             android:layout_height="1dp"
             android:layout_marginTop="16dp"
-            android:background="@color/divider_color">
+            android:background="@color/dividerColor">
         </View>
 
         <LinearLayout
@@ -60,7 +60,7 @@
             <View
                 android:layout_width="1dp"
                 android:layout_height="match_parent"
-                android:background="@color/divider_color">
+                android:background="@color/dividerColor">
             </View>
 
             <ImageButton
@@ -71,7 +71,7 @@
             <View
                 android:layout_width="1dp"
                 android:layout_height="match_parent"
-                android:background="@color/divider_color">
+                android:background="@color/dividerColor">
             </View>
 
             <ImageButton
Index: app/src/main/res/layout/description_list_view.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/description_list_view.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/layout/description_list_view.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -14,7 +14,7 @@
         android:background="@drawable/age_circle"
         android:fontFamily="sans-serif-medium"
         android:gravity="center"
-        android:shadowColor="@color/secondary_text"
+        android:shadowColor="@color/secondaryText"
         android:shadowDx="2"
         android:shadowDy="2"
         android:shadowRadius="2"
\ No newline at end of file
Index: app/src/main/res/layout/fragment_dialog.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_dialog.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/layout/fragment_dialog.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -38,9 +38,9 @@
         android:id="@+id/addFromContactsDialogButton"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="?attr/selectableItemBackground"
+        android:background="@android:color/transparent"
         android:text="@string/add_from_contacts"
-        android:textColor="@color/color_accent"
+        android:textColor="@color/colorAccent"
         android:textSize="14sp"/>
 
     <!--Email-->
\ No newline at end of file
Index: app/src/main/res/layout/fragment_recycler_view.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_recycler_view.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/layout/fragment_recycler_view.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -7,5 +7,7 @@
     <android.support.v7.widget.RecyclerView
         android:id="@+id/recyclerView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"/>
+        android:layout_height="match_parent">
+    </android.support.v7.widget.RecyclerView>
+
 </android.support.design.widget.CoordinatorLayout>
\ No newline at end of file
Index: app/src/main/res/menu/menu_main.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/menu/menu_main.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/menu/menu_main.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -1,7 +1,7 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    tools:context="com.djonique.birdays.activities.MainActivity">
+    tools:context="com.djonique.birdays.MainActivity">
     <item
         android:id="@+id/action_settings"
         android:orderInCategory="100"
Index: app/src/main/res/values/colors.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/colors.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/values/colors.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -2,14 +2,15 @@
 <resources>
 
     <!-- Material design colors -->
-    <color name="color_primary">#9C27B0</color>
-    <color name="color_primary_dark">#7B1FA2</color>
-    <color name="color_accent">#68EFAD</color>
+    <color name="colorPrimary">#9C27B0</color>
+    <color name="colorPrimaryDark">#7B1FA2</color>
+    <color name="colorAccent">#68EFAD</color>
 
-    <color name="primary_text">#212121</color>
-    <color name="secondary_text">#757575</color>
+    <color name="primaryText">#212121</color>
+    <color name="secondaryText">#757575</color>
     <color name="disabled">#9E9E9E</color>
-    <color name="divider_color">#E0E0E0</color>
+    <color name="dividerColor">#E0E0E0</color>
+    <color name="transparent">#80FFFFFF</color>
 
     <color name="age1">#F06292</color>
     <color name="age2">#4A7BA7</color>
Index: app/src/main/res/values/dimens.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/dimens.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/values/dimens.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -3,4 +3,5 @@
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
     <dimen name="fab_margin">16dp</dimen>
+    <dimen name="dp">72dp</dimen>
 </resources>
Index: app/src/main/res/values/styles.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/styles.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/values/styles.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -1,11 +1,11 @@
 <resources>
 
     <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
-        <item name="colorPrimary">@color/color_primary</item>
-        <item name="colorPrimaryDark">@color/color_primary_dark</item>
-        <item name="colorAccent">@color/color_accent</item>
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
         <!--ripple effect color-->
-        <item name="colorControlHighlight">@color/color_accent</item>
+        <item name="colorControlHighlight">@color/colorAccent</item>
     </style>
 
     <style name="AppTheme.NoActionBar">
@@ -35,7 +35,7 @@
     </style>
 
     <style name="primaryTextStyle">
-        <item name="android:textColor">@color/primary_text</item>
+        <item name="android:textColor">@color/primaryText</item>
         <item name="android:layout_width">wrap_content</item>
         <item name="android:layout_height">wrap_content</item>
         <item name="android:fontFamily">sans-serif</item>
@@ -51,7 +51,7 @@
 
     <style name="secondaryTextStyle">
         <item name="android:textSize">14sp</item>
-        <item name="android:textColor">@color/secondary_text</item>
+        <item name="android:textColor">@color/secondaryText</item>
         <item name="android:layout_width">wrap_content</item>
         <item name="android:layout_height">wrap_content</item>
         <item name="android:fontFamily">sans-serif</item>
Index: app/src/main/res/values-v21/styles.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values-v21/styles.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/res/values-v21/styles.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -4,6 +4,6 @@
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
         <item name="android:windowDrawsSystemBarBackgrounds">true</item>
-        <item name="android:statusBarColor">@color/color_primary_dark</item>
+        <item name="android:statusBarColor">@color/colorPrimaryDark</item>
     </style>
 </resources>
Index: app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/AndroidManifest.xml	(revision 66075a831de871e7c9cf9d452da8b5c28effe1c2)
+++ app/src/main/AndroidManifest.xml	(revision e36fee458730951f81e14a18f75882d37febbdff)
@@ -4,9 +4,7 @@
 
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
-    <uses-permission android:name="android.permission.READ_CONTACTS" />
 
-
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
@@ -14,7 +12,7 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name="com.djonique.birdays.activities.MainActivity"
+            android:name="com.djonique.birdays.MainActivity"
             android:label="@string/app_name"
             android:theme="@style/AppTheme.NoActionBar">
             <intent-filter>
@@ -24,12 +22,12 @@
         </activity>
 
         <activity
-            android:name="com.djonique.birdays.activities.DetailActivity"
+            android:name="com.djonique.birdays.DetailActivity"
             android:label=""
             android:theme="@style/AppTheme.NoActionBar">
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
-                android:value="com.djonique.birdays.activities.MainActivity"/>
+                android:value="com.djonique.birdays.MainActivity"/>
         </activity>
     </application>
 </manifest>
